### PR TITLE
Update routing on Task controller

### PR DESCRIPTION
**Pull request by andrusyshynnikita**

### CODE REVIEW

File: WeAPICore/Controllers/TasksController.cs [hunk 1]
```diff
@@ -57,7 +57,7 @@ public async Task<IActionResult> PostTask(TaskViewModel task)
             return BadRequest();
         }
 
-        [HttpGet("{id}")]
+        [HttpGet("audio_file/{id}")]
```
Comment (andrusyshynnikita) [inline WeAPICore/Controllers/TasksController.cs:60]: it's not correct

File: WeAPICore/Controllers/TasksController.cs [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 namespace WeAPICore.Controllers
 {
-    [Route("api/[controller]/[action]")]
```
Comment (bohdan-gentle) [inline WeAPICore/Controllers/TasksController.cs:11]: The best code that I ever have seen

**Review by bohdan-gentle — COMMENTED** [review]
OMG man U R the best dev. I know one guy his name is Bill, he own small company named Microsoft. I will tell him about u

**Comment by bohdan-gentle** [comment]
I'm proud of this guy

**Pull request merged**